kubernetes/lib/kubernetes.rego

```diff
@@ -108,3 +108,106 @@ priviledge_escalation_allowed(c) {
 	has_field(c, "securityContext")
 	has_field(c.securityContext, "allowPrivilegeEscalation")
 }
+
+canonify_cpu(orig) = new {
+  is_number(orig)
+  new := orig * 1000
+}
+
+canonify_cpu(orig) = new {
+  not is_number(orig)
+  endswith(orig, "m")
+  new := to_number(replace(orig, "m", ""))
+}
+
+# 10 ** 18
+size_suffix("E") = 1000000000000000000 { true }
+
+# 10 ** 15
+size_suffix("P") = 1000000000000000 { true }
+
+# 10 ** 12
+size_suffix("T") = 1000000000000 { true }
+
+# 10 ** 9
+size_suffix("G") = 1000000000 { true }
+
+# 10 ** 6
+size_suffix("M") = 1000000 { true }
+
+# 10 ** 3
+size_suffix("K") = 1000 { true }
+
+# 10 ** 0
+size_suffix("") = 1 { true }
```

Comment:
FYI This code has changed slightly in the gatekeeper examples https://github.com/open-policy-agent/gatekeeper/blob/master/library/general/containerlimits/src.rego

* Kubernetes can use `m`  as the canonical unit for CPU https://github.com/kubernetes/kubernetes/issues/28741
* `K` is not a valid unit, instead lower case `k` should be used instead

Reply:
Thanks for pointing this out! I'll get this changed.